### ISSUE TITLE
CanvasRenderingContext2D::closePath()     LinkError: import object field 'closePath' is not a Function

### ISSUE THREAD

**Issue by WhiteHexagon** [mirkosertic/Bytecoder#353]
Apologies if this has been fixed in a newer version, but I switched back to June'19 cli build so that I could make some progress using Kotlin.

The other canvas2D commands I have tried are working nice so far :) until I tried:
`ctx.closePath()`

resulting in a browser console message:
`LinkError: import object field 'closePath' is not a Function`

I tried an example on w3schools site that makes use of closePath, and it seems to work, so I am fairly confident it is not an issue with my version of firefox.  Any ideas please?

**Comment by WhiteHexagon** [comment]
As an additional point on the same topic, in my custom class that extends `CanvasRenderingContext2D` I tried to add a couple other functions:

```
fun quadraticCurveTo(cx: Int, cy: Int, dx: Int, dy: Int)
//fun strokeText(text: String, x: Int, y: Int)
    fun strokeText(text: String, x: Float, y: Float)
```

I can't seem to get the types right, and get similar errors to the above:

`LinkError: import object field 'strokeTextStringINTINT' is not a Function`

I can see in the implementation that there are different types being used for different methods:

```
    void lineTo(float v, float v1);
    void arc(double v, double v1, double v2, double v3, double v4, boolean b);
```

but I cannot work out where this information is coming from.  I checked a few canvas2D documents, and none of them seem to specify types of the parameters.  So I am wondering how do I work out the correct signature when adding new function mappings?
